test/sql/append.sql

```diff
@@ -0,0 +1,74 @@
+\ir include/create_single_db.sql
+
+-- create a now() function for repeatable testing that always returns
+-- the same timestamp. It needs to be marked STABLE
+CREATE OR REPLACE FUNCTION now_s()
+RETURNS timestamp LANGUAGE PLPGSQL STABLE AS
+$BODY$
+BEGIN
+     RETURN '2017-08-22T10:00:00'::timestamp;
+END;
+$BODY$;
+
+CREATE OR REPLACE FUNCTION now_i()
+RETURNS timestamp LANGUAGE PLPGSQL IMMUTABLE AS
+$BODY$
+BEGIN
+     RETURN '2017-08-22T10:00:00'::timestamp;
+END;
+$BODY$;
+
```

Comment:
So what I am worried about is executing VOLATILE functions more often than plain sql would expect. Since VOLATILE functions modify data, isn't that potentially a correctness violations?
Or am I missing something?